modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,106 @@
+         $('.secondary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.content-tabs a[aria-selected]').length === 0) {
+         $('.content-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     }
+     // else if ($('.vertical-tabs a[aria-selected]').length === 0) {
+     //     $('.vertical-tabs a:first').attr({'aria-selected':'true', 'tabindex':'0'});
+     //     $('.vertical-tabs ul').attr('aria-orientation':'vertical');
+     // }
+
+     /* Click on a link */
+     $('body').on('click', '.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs
+
+         let $target = $(event.target);
+         let tabNameClick = '';
+
+         if ($target.hasClass('master-tabs')) {
+             tabNameClick = '.master-tabs';
+         } else if ($target.hasClass('primary-tabs')) {
+             tabNameClick = '.primary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameClick = '.secondary-tabs';
+         } else if ($target.hasClass('content-tabs')) {
+             tabNameClick = '.content-tabs';
+         }
+         // else if($target.hasClass('vertical-tabs')) {
+         //     tabNameClick = '.vertical-tabs';
+         // }
+
+         // remove aria selected on all links + remove focusable
+         $(tabNameClick + ' a').attr('aria-selected', 'false');
+
+         // add aria selected on $this + focusable
+         $(tabNameClick + ' a.focus-visible').attr('aria-selected', 'true');
+
+         // Important - Must be set to true for October to set class="active" in the <li>
+         return true;
+     });
+
+     /* Keyboard arrow selection */
+     $('body').on('keydown', '.primary-tabs,.secondary-tabs,.master-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs
+
+         let $target = $(event.target);
+         let tabNameKey = '';
+
+         if ($target.hasClass('primary-tabs')) {
+             tabNameKey = '.primary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameKey = '.secondary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameKey = '.secondary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameKey = '.content-tabs';
+         }
+         // else if($target.hasClass('vertical-tabs')) {
+         //     tabNameKey = '.vertical-tabs';
+         // }
+
+         let strikeUpOrRightTab = event.key === 'ArrowLeft' || event.key === 'ArrowUp';
+         let strikeDownOrLeftTab = event.key === 'ArrowDown' || event.key === 'ArrowRight';
+         if (strikeUpOrRightTab || strikeDownOrLeftTab) {
+             event.preventDefault();
+
+             var postition = strikeUpOrRightTab ? 'first-child' : 'last-child';
+             $activated = $(tabNameKey + ' a[aria-selected="true"]').parent();
```

Comment:
@ayumihamsaki I think, you've missed var here?

Reply:
patched.